pkg/controllers/management/cloudcredential/controller.go

```diff
@@ -44,10 +44,11 @@ func (n *Controller) ccSync(key string, cloudCredential *v1.Secret) (runtime.Obj
 		return cloudCredential, fmt.Errorf("cloud credential %v has no creatorId annotation", cloudCredential.Name)
 	}
 	if err := globalnamespacerbac.CreateRoleAndRoleBinding(
-		globalnamespacerbac.CloudCredentialResource, cloudCredential.Name, cloudCredential.UID, []v3.Member{}, creatorID,
-		n.managementContext, "*"); err != nil {
+		globalnamespacerbac.CloudCredentialResource, cloudCredential.Name, "v1", creatorID, []string{"*"}, cloudCredential.UID, []v3.Member{},
+		n.managementContext); err != nil {
```

Comment:
I did not get this change for On Edit Cluster, Answers can be updated, please can you explain a little

Reply:
@prachidamle not sure I get the `Answers` you're referring to? I just changed the args of the method. Previously for cloud credentials it was setting apiversion to `v1` and apigroup to ['*'], so instead I'm passing it from this method

Reply:
Answers was a typo. Updated